src/overlays/actors/ovl_Dm_Gm/z_dm_gm.c

```diff
@@ -31,37 +33,336 @@ const ActorInit Dm_Gm_InitVars = {
+    DmGm* this = THIS;
+    s16 phi_v1;
+    s16 phi_v0;
+
+    if (!(this->unk_2AE & 1)) {
+        if (this->unk_2AE & 2) {
+            phi_v0 = 1;
+        } else {
+            phi_v0 = 0;
+        }
+        phi_v1 = 1;
+    } else {
+        phi_v1 = 0;
+        phi_v0 = 0;
+    }
+
+    if (limbIndex == OBJECT_AN1_LIMB_09) {
+        func_8013AD9C(this->unk_2BE + this->unk_2C2 + 0x4000,
+                      this->unk_2C0 + this->unk_2C4 + this->actor.shape.rot.y + 0x4000, &this->unk_18C, &this->unk_1A4,
+                      phi_v1, phi_v0);
+        Matrix_StatePop();
+        Matrix_InsertTranslation(this->unk_18C.x, this->unk_18C.y, this->unk_18C.z, MTXMODE_NEW);
+        Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
+        Matrix_RotateY(this->unk_1A4.y, MTXMODE_APPLY);
+        Matrix_InsertXRotation_s(this->unk_1A4.x, MTXMODE_APPLY);
+        Matrix_InsertZRotation_s(this->unk_1A4.z, MTXMODE_APPLY);
+        Matrix_StatePush();
+    } else if (limbIndex == OBJECT_AN1_LIMB_02) {
+        func_8013AD9C(this->unk_2C2 + 0x4000, this->unk_2C4 + this->actor.shape.rot.y + 0x4000, &this->unk_194,
+                      &this->unk_1AA, phi_v1, phi_v0);
+        Matrix_StatePop();
+        Matrix_InsertTranslation(this->unk_194.x, this->unk_194.y, this->unk_194.z, MTXMODE_NEW);
+        Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
+        Matrix_RotateY(this->unk_1AA.y, MTXMODE_APPLY);
+        Matrix_InsertXRotation_s(this->unk_1AA.x, MTXMODE_APPLY);
+        Matrix_InsertZRotation_s(this->unk_1AA.z, MTXMODE_APPLY);
+        Matrix_StatePush();
+    }
+}
+
+TexturePtr D_80C25238[] = {
+    object_an1_Tex_00E6E0,
+    &object_an1_Tex_00F7A0,
+    &object_an1_Tex_0101A0,
+};
+
+TexturePtr D_80C25244[] = {
+    &object_an1_Tex_00E1E0, &object_an1_Tex_00EFA0, &object_an1_Tex_00F3A0, &object_an1_Tex_00EFA0,
+    &object_an1_Tex_00FDA0, &object_an1_Tex_00F9A0, &object_an1_Tex_0103A0,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Gm/DmGm_Init.s")
+void func_80C25000(Actor* thisx, GlobalContext* globalCtx) {
+    DmGm* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Gm/DmGm_Destroy.s")
+    OPEN_DISPS(globalCtx->state.gfxCtx);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Gm/DmGm_Update.s")
+    func_8012C28C(globalCtx->state.gfxCtx);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Gm/func_80C24CD0.s")
+    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(D_80C25244[this->unk_2B8]));
+    gSPSegment(POLY_OPA_DISP++, 0x09, Lib_SegmentedToVirtual(D_80C25238[0]));
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Gm/func_80C24E4C.s")
+    SkelAnime_DrawTransformFlexOpa(globalCtx, this->skelAnime.skeleton, this->skelAnime.jointTable,
+                                   this->skelAnime.dListCount, NULL, func_80C24CD0, func_80C24E4C, &this->actor);
```

Comment:
```suggestion
                                   this->skelAnime.dListCount, NULL, func_80C24CD0, func_80C24E4C, &this->actor);
```
Looks like func_80C24CD0 is the PostLimb and func_80C24E4C the transformLimb